core/src/Kokkos_Serial.hpp

```diff
@@ -90,10 +91,10 @@ class Serial {
 
   //! Tag this class as an execution space:
   using execution_space = Serial;
-  //! The size_type alias best suited for this device.
-  using size_type = HostSpace::size_type;
   //! This device's preferred memory space.
-  using memory_space = HostSpace;
+  using memory_space = Kokkos::Impl::DefaultHostMemorySpace;
```

Comment:
we may want to make DefaultHostMemorySpace be part of Kokkos:: not Kokkos::Impl:: same as DefaultHostExecutonSpace